Assets/UniversalRenderPipelineGlobalSettings.asset.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
and this file?

Reply:
Thought it was a file that had been left out by the gitignore change, but its not after all. Just pushed a fix